### PR TITLE
Added Fuse Runtime to the homepage.

### PR DESCRIPTION
**Pull request by valdar**

### CODE REVIEW

File: src/app/wizard/pages/intro/intro.component.html [hunk 1]
```diff
@@ -23,8 +23,8 @@ <h1>Supported Runtimes</h1>
       <div class="col-sm-3 runtime">
```
Comment (edewit) [inline src/app/wizard/pages/intro/intro.component.html]: we are using bootstrap grid system there are 12 columns now there are 15. see https://getbootstrap.com/docs/3.3/css/#grid-options

**Comment by edewit** [comment]
I think it would be better to split them over 2 rows put swarm, vertx and boot in the first row and then node and fuse in the second using offsets to position them in the center

File: src/app/wizard/pages/intro/intro.component.html (hunk 1 above)
Comment (valdar) [reply src/app/wizard/pages/intro/intro.component.html]: what do you suggest? `class="col-sm-2"` for all?

Comment (edewit) [reply src/app/wizard/pages/intro/intro.component.html]: better to split it into 2 rows

**Comment by valdar** [comment]
I see, how I'd like to centre the 3 runtimes on the top row? the are 3 col each so we have 3 col left do you like 1 on the left and 2 on the right or the other way around. or should we resize each runtime to something different from 3 col?

**Comment by aileenc** [comment]
This PR is merged but we don't show up yet in launch.openshift.io - is there another step?

**Comment by quintesse** [comment]
@aileenc first of all the correct url is https://developers.redhat.com/launch

and second, that url points to the production environment where it will only appear after a new release train has been done. So if you want to see the result of your PR you need to go to the staging environment: https://launch.prod-preview.openshift.io/wizard

But you can't see it there either because the build failed for some reason: https://ci.centos.org/view/Devtools/job/devtools-launcher-frontend-generator-build-master/24/console

Any idea what could be wrong @edewit ? It doesn't really seem to have to do anything with problems in the code.

**Comment by quintesse** [comment]
@aileenc FYI you can see the result now on https://launch.prod-preview.openshift.io

**Comment by aileenc** [comment]
@quintesse Thanks, I see it now on staging.

When is the next release train to get this on production?

Also, if we want to add more Fuse boosters can we only do this when there is a release train?

**Comment by quintesse** [comment]
@aileenc it should be some time this week.

You can add boosters whenever you want, although until we get a proper release procedure in place it's best to make sure that the ones you add are ready to be put into production at any time without prior warning.

**Pull request merged**